backend/Dockerfile

```diff
@@ -15,7 +15,8 @@ RUN pip-compile requirements.in > requirements.txt && \
     pip-compile dev-requirements.in > dev-requirements.txt
 
 RUN pip install -r requirements.txt && \
-    pip install -r dev-requirements.txt
+    pip install -r dev-requirements.txt  && \
+    pip install psycopg2-binary
```

Comment:
Can we add it directly to `requirements.in`?

Reply:
Any chance it could impact on Heroku's deployment? I've seen a project that has a `--no-binary psycopg2` flag on the deployment requirements file.

Reply:
you're right, it's not recommended for production